sktime/forecasting/naive.py

```diff
@@ -70,42 +79,40 @@ def fit(self, y_train, fh=None, X_train=None):
         self._set_oh(y_train)
         self._set_fh(fh)
 
-        if self.strategy in ("last", "seasonal_last"):
+        if self.strategy == "last":
             if self.window_length is not None:
-                warn("For the `last` and `seasonal_last` strategy, "
+                warn("For the `last` strategy, "
                      "the `window_length_` value will be ignored.")
 
-        if self.strategy in ("last", "mean"):
-            if self.sp is not None:
-                warn("For the `last` and `mean` strategy, "
-                     "the `sp` value will be ignored.")
-
-        if self.strategy == "last":
+        if self.strategy == "last" and self.sp is None:
```

Comment:
I believe the default argument for sp should be 1, not None. Could you change that here and throughout the code? So if sp is 1, we basically don't have seasonality.